RIOT-OS/RIOT
# github: add 'testing' to the PR template

### Contribution description

Add a section on the testing procedure for a pull request.

It should help reviewing pull requests as they often not describe which application raises the error they are fixing or any way of verifying that a feature is correctly implemented.


I often add this section myself in pull requests and I think it would benefit if it was by default.

### Testing procedure (EDIT)

It should be used by github for showing the result but you can review the raw content:

* Check the files changed
* Click 'View' on the `PULL_REQUEST_TEMPLATE.md` file
* Click 'Raw'

You should see what will be displayed on a PR.

For the current state you can check the text displayed on a new pull request submission.

### Issues/PRs references

No specific ones
